### DISCUSSION TITLE
Deployment on GCP questions

### DISCUSSION THREAD

**Discussion by busetde** [SUSE/ha-sap-terraform-deployments#667] [Q&A] [answered]
Hi,

I'm trying to run it on GCP and the deployment progress at SAP NW.
Could anyone advise in regards to the error below:

![image](https://user-images.githubusercontent.com/63712632/112796219-e1ee1180-9093-11eb-8f72-e10f38337447.png)

Regards - Budi

**Comment by yeoldegrove** [comment, marked as answer]
This was totally unnoticed. Feel free to bump us if you still have issues.

**Reply by busetde** [reply]
Apologise, please kindly close this... 

Thanks - Budi
